plugin.json

```diff
@@ -91,6 +91,15 @@
                 "regenerate_help_text": "",
                 "placeholder": "",
                 "default": null
+            },
+            {
+                "key": "RestrictBotPosting",
+                "display_name": "Restrict Bot Posting:",
```

Comment:
It's not the bot who posts the meeting. The meeting is created by the user. We should update this wherever applicable.